catalog/transformer/catalog-transformer-geojson-queryresponse/src/test/java/ddf/catalog/transformer/queryresponse/geojson/TestGeoJsonQueryResponseTransformer.java

```diff
@@ -70,9 +70,24 @@
 
   private static final Date NOW = new Date();
 
+  private GeoJsonQueryResponseTransformer geoJsonQueryResponseTransformer;
+
+  public TestGeoJsonQueryResponseTransformer() {
```

Comment:
I think the convention is to use `@Before`

Reply:
The way I understand it, `@BeforeClass` executes the code in the annotated method once at the start of the unit test group while `@Before` executes the code in the annotated method before the start of each test method. I'm not sure when or how this constructor is called; if each unit test is executed from a new `TestGeoJsonQueryResponseTransformer`, then `@Before` would be appropriate, but if all the unit tests are run from one `TestGeoJsonQueryResponseTransformer` (which is what I would guess), then `@BeforeClass` would be appropriate.

That ambiguity makes me further question why a constructor is used in a test class rather than a `@Before` or `@BeforeClass` method.